assets/extensions/login-template.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!--
+
+This template can be modified and used to customize the login page. To replace
+the login page, set master configuration option oauthConfig.templates.login to
+the path of the template file. Do not change lines with curly braces {{ }} below.
```

Comment:
I'll soften the wording. Some changes would break things even if you leave the tokens, and our validation wouldn't catch it. Admittedly you can leave those lines alone and still break things.